src/Sarif/Sarif.csproj

```diff
@@ -42,15 +42,16 @@
     <!--
     Marking these items as content causes them to be included in the NuGet package.
     -->
-    <Content Include="Schemata\sarif-schema.json" />
+    <Content Include="Schemata\sarif-schema-2.1.0-rtm.0.json" />
```

Comment:
>sarif-schema-2.1.0-rtm.0.json [](start = 31, length = 29)

Note how many different places this appears. We must either DRY it out or go through this again if we need rtm.1.